File: python/examples/projects.py
from aristech_nlp_client import NlpClient
from utils import host, auth_token, auth_secret, root_cert, ssl

client = NlpClient(host=host, ssl=ssl, root_cert=root_cert, auth_token=auth_token, auth_secret=auth_secret)
projects = client.list_projects()
for project in projects:
    print(project)

File: rust/examples/projects.rs
mod utils;
use std::error::Error;
use utils::get_tls_options;

use aristech_nlp_client::{get_client, list_projects};

#[tokio::main]
async fn main() -> Result<(), Box<dyn Error>> {
    // Load environment variables from .env file
    dotenv::dotenv().ok();

    let host = std::env::var("HOST")?;
    let tls_options = get_tls_options()?;
    let mut client = get_client(host, tls_options).await?;

    let projects = list_projects(&mut client, None).await?;
    for project in projects {
        println!("{:?}", project);
    }
    Ok(())
}
